test/import/FSHImporter.Profile.test.ts

```diff
@@ -0,0 +1,528 @@
+        const profile = result.profiles.get('ObservationProfile');
+        expect(profile.rules).toHaveLength(1);
+        assertFixedValueRule(profile.rules[0], 'valueString', 'hello\nworld');
+      });
+
+      it('should parse fixed value date rule', () => {
+        const input = `
+        Profile: ObservationProfile
+        Parent: Observation
+        * valueDateTime = 2019-11-01T12:30:01.999Z
+        `;
+
+        const result = importText(input);
+        const profile = result.profiles.get('ObservationProfile');
+        expect(profile.rules).toHaveLength(1);
+        // For now, treating dates like strings
+        assertFixedValueRule(profile.rules[0], 'valueDateTime', '2019-11-01T12:30:01.999Z');
+      });
+
+      it('should parse fixed value time rule', () => {
+        const input = `
+        Profile: ObservationProfile
+        Parent: Observation
+        * valueTime = 12:30:01.999-05:00
+        `;
+
+        const result = importText(input);
+        const profile = result.profiles.get('ObservationProfile');
+        expect(profile.rules).toHaveLength(1);
+        // For now, treating dates like strings
+        assertFixedValueRule(profile.rules[0], 'valueTime', '12:30:01.999-05:00');
+      });
+
+      it('should parse fixed value code rule', () => {
+        const input = `
+        Alias: LOINC = http://loinc.org
+
+        Profile: ObservationProfile
+        Parent: Observation
+        * status = #final
+        `;
+
+        const result = importText(input);
+        const profile = result.profiles.get('ObservationProfile');
+        expect(profile.rules).toHaveLength(1);
+        assertFixedValueRule(profile.rules[0], 'status', new Code('final'));
+      });
+
+      it('should parse fixed value CodeableConcept rule', () => {
+        const input = `
+        Alias: LOINC = http://loinc.org
+
+        Profile: ObservationProfile
+        Parent: Observation
+        * valueCodeableConcept = LOINC#718-7 "Hemoglobin [Mass/volume] in Blood"
+        `;
+
+        const result = importText(input);
+        const profile = result.profiles.get('ObservationProfile');
+        expect(profile.rules).toHaveLength(1);
+        assertFixedValueRule(
+          profile.rules[0],
+          'valueCodeableConcept',
+          new Code('718-7', 'http://loinc.org', 'Hemoglobin [Mass/volume] in Blood')
+        );
+      });
+
+      it('should parse fixed value Quantity rule', () => {
+        const input = `
+
+        Profile: ObservationProfile
+        Parent: Observation
+        * valueQuantity = 1.5 'mm'
+        `;
+
+        const result = importText(input);
+        const profile = result.profiles.get('ObservationProfile');
+        expect(profile.rules).toHaveLength(1);
+        assertFixedValueRule(
+          profile.rules[0],
+          'valueQuantity',
+          new Quantity(1.5, new Code('mm', 'http://unitsofmeasure.org'))
+        );
+      });
+
+      it('should parse fixed value Ratio rule', () => {
```

Comment:
In the spirit of lots of coverage, we could add a test for when a ratio is just a number, and not a `Quantity`.

Reply:
Yeah, totally should.  I think it was like 1:30am when I was writing these tests.  Subconsciously, I was probably just trying to get to bed as soon as I could!